packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -1,98 +1,49 @@
-import React, { ComponentProps } from "react";
+import React, { ComponentProps, useState } from "react";
 
+import { classNames } from "@calcom/lib";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Icon } from "@calcom/ui/Icon";
 
-import { Icon } from "../../../Icon";
 import { useMeta } from "../Meta";
 import Shell from "../Shell";
-import VerticalTabs, { VerticalTabItem } from "../navigation/tabs/VerticalTabs";
-
-const tabs = [
```

Comment:
Why?

Reply:
I moved it to `packages/ui/v2/core/navigation/SettingsSidebarContainer.tsx` since I was thinking that would be the only time we would need to access those tabs.

Reply:
Id rather keep this outside of react since they should be "static"